modules/alloy/src/test/ts/browser/ui/dropdown/TieredMenuHighlightOnOpenTest.ts

```diff
@@ -0,0 +1,172 @@
+
+    it('Test basic structure, before and after highlightPrimary', () => {
+      const tmenuComp = hook.component();
+      Assertions.assertStructure(
+        'Testing the Tiered Menu structure',
+        ApproxStructure.build((s, str, arr) => {
+          return s.element('div', {
+            classes: [ ],
+            children: [
+              s.element('ol', {
+                // The menu should *not* be highlighted
+                classes: [ arr.has('menu'), arr.not('selected-menu') ],
+                children: [
+                  TestDropdownMenu.structNotActiveItem,
+                  TestDropdownMenu.structNotActiveItem,
+                  TestDropdownMenu.structNotActiveItem
+                ]
+              })
+            ]
+          });
+        }),
+        tmenuComp.element
+      );
+
+      TieredMenu.highlightPrimary(tmenuComp);
+      Assertions.assertStructure(
+        'Testing the Tiered Menu structure after highlightPrimary',
+        structActiveMenuAndItem,
+        tmenuComp.element
+      );
+    });
+  });
+
+  context('HighlightOnOpen = HighlightJustMenu', () => {
+    const hook = GuiSetup.bddSetup(
+      (store) => GuiFactory.build(
+        makeTieredMenu(store, HighlightOnOpen.HighlightJustMenu)
+      )
+    );
+
+    it('Test basic structure', () => {
+      const tmenuComp = hook.component();
+      Assertions.assertStructure(
+        'Testing the Tiered Menu structure',
+        ApproxStructure.build((s, str, arr) => {
+          return s.element('div', {
+            classes: [ ],
+            children: [
+              s.element('ol', {
+                // The menu should be highlighted
+                classes: [ arr.has('menu'), arr.has('selected-menu') ],
+                children: [
+                  TestDropdownMenu.structNotActiveItem,
+                  TestDropdownMenu.structNotActiveItem,
+                  TestDropdownMenu.structNotActiveItem
+                ]
+              })
+            ]
+          });
+        }),
+        tmenuComp.element
+      );
+
+      TieredMenu.highlightPrimary(tmenuComp);
+      Assertions.assertStructure(
+        'Testing the Tiered Menu structure after highlightPrimary',
+        structActiveMenuAndItem,
+        tmenuComp.element
+      );
+    });
+  });
+
+  context('HighlightOnOpen = HighlightMenuAndItem', () => {
+    const hook = GuiSetup.bddSetup(
+      (store) => GuiFactory.build(
+        makeTieredMenu(store, HighlightOnOpen.HighlightMenuAndItem)
+      )
+    );
+
+    it('Test basic structure', () => {
+      const tmenuComp = hook.component();
+      Assertions.assertStructure(
+        'Testing the Tiered Menu structure',
+        structActiveMenuAndItem,
+        tmenuComp.element
+      );
+
+      TieredMenu.highlightPrimary(tmenuComp);
+      Assertions.assertStructure(
+        'Testing the Tiered Menu structure after highlightPrimary',
+        structActiveMenuAndItem,
+        tmenuComp.element
+      );
+    });
+  });
+});
```

Comment:
I think you need to setup your IDE to use the `EditorConfig` plugin... many of these new files are missing a blank line.

Reply:
K. Yeah, I didn't have that plugin. 

Is there a command I can run to reformat all files according to the EditorConfig?

Reply:
Also, I've been running eslint over these changes and it hasn't complained. Can a trailing newline be encompassed in an eslint rule?